source/codegen/common_helpers.py

```diff
@@ -13,6 +13,9 @@ def is_enum(parameter):
 def is_struct(parameter):
   return parameter["type"].startswith("struct")
 
+def is_byte_type(dataType):
+  return dataType == 'ViUInt8[]'
```

Comment:
In future, if more similar DataTypes come, then it can be changed to

```
def is_byte_type(dataType) :
     return dataType in ['ViUInt8', new_data_type]
```